app/src/main/java/org/mozilla/fenix/onboarding/JumpBackInCFRDialog.kt

```diff
@@ -27,7 +27,7 @@ class JumpBackInCFRDialog(val recyclerView: RecyclerView) {
     /**
      * Try to show the crf dialog if it hasn't been shown before.
      */
-    fun showIfNeeded() {
+    fun showIfNeeded(): Boolean {
```

Comment:
This change could be reverted if the suggestion to switch to a `when` statement is taken.